src/Command/ImageRestoreCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace RokkaCli\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ImageRestoreCommand extends BaseRokkaCliCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('image:restore')
+            ->setDescription('Restore the given image from Rokka')
```

Comment:
is "from" correct? maybe just "Restore an image"?

Reply:
removed. everywhere